devel/hg-keyring/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                mercurial_keyring
+version             1.4.0
+
+name                hg-keyring
+
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             BSD
+maintainers         {danchr @danchr} openmaintainer
+
+description         Mercurial extension used to securely save passwords
+long_description \
+    mercurial_keyring is a Mercurial extension used to securely save HTTP and \
+    SMTP authentication passwords in password databases, including the macOS \
+    Keychain.
+
+checksums           rmd160  f5ccc126edf4213c0b1197d1d32a975358a1659b \
+                    sha256  b835d5dcd86c2de58954e93d9864473833f83627391cffa8654b7fa49d572288 \
+                    size    27914
+
+python.default_version  39
+python.rootname         mercurial_keyring
+
+depends_lib-append      port:mercurial
+
+notes               "
+To enable the keyring extension, add the following to your ~/.hgrc:
+
+\[extensions\]
+mercurial_keyring =
+"
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+depends_run-append \
+                    port:py${python.version}-keyring \
+                    port:py${python.version}-mercurial_extension_utils \
+                    port:mercurial
```

Comment:
does `mercurial` need to be both `depends_lib` and `depends_run` - that seems redundant, right?